src/algebra/geom_sum.lean

```diff
@@ -197,6 +197,20 @@ lemma mul_neg_geom_sum [ring α] (x : α) (n : ℕ) :
   (1 - x) * (∑ i in range n, x ^ i) = 1 - x ^ n :=
 op_injective $ by simpa using geom_sum_mul_neg (op x) n
 
+lemma commute.geom_sum₂_comm {α : Type u} [semiring α] (x y : α) (n : ℕ) (h : commute x y) :
```

Comment:
I guess there is an `{α : Type u} [semiring α]` already defined.
```suggestion
lemma commute.geom_sum₂_comm {α : Type u} [semiring α] {x y : α} (n : ℕ) (h : commute x y) :
```
Also, looking at other `commute` lemmas in this file, I think you should add the `protected` attribute.